opentracing/event.go

```diff
@@ -0,0 +1,22 @@
+package opentracing
+
+import (
+	"time"
+
+	"sourcegraph.com/sourcegraph/appdash"
+)
+
+func init() {
+	appdash.RegisterEvent(spanCompletionEvent{})
```

Comment:
This should be removed and possibly expose something similar to this in Appdash.